src/lib/kwlib.sh

```diff
@@ -126,6 +127,21 @@ function show_verbose()
   [[ "$flag" == 'VERBOSE' ]] && say "$cmd"
 }
 
+# This function create the shared memory directory
```

Comment:
Technically, you added a fallback behavior that creates the folder in the `/tmp`; you must highlight this in the doc. Also, add an empty line between the doc description and the parameter.